tools/db_stress.cc

```diff
@@ -2360,8 +2360,8 @@ int main(int argc, char** argv) {
   SetUsageMessage(std::string("\nUSAGE:\n") + std::string(argv[0]) +
                   " [OPTIONS]...");
   ParseCommandLineFlags(&argc, &argv, true);
-#if !defined(OS_MACOSX) && !defined(OS_WIN) && !defined(OS_SOLARIS) &&  \
```

Comment:
@lightmark since you know this code the best, what do you suggest we do here? 
Shall I go ahead with this commit, and may be, you might want to address Andrew's concern separately?